File: kotlinlib/src/main/java/com/example/lib/TestAct_.java
package com.example.lib;

import android.app.Activity;
import android.view.View;



import kotlin.Lazy;
import kotlin.LazyKt;

public class TestAct_ extends Activity {

    private final Lazy<View> delegate = this.bindView(this, R.layout.activity_main);

    private View getMBtnStartHostRegisterAct() {
        return (View) this.delegate.getValue();
    }

    private Lazy<View> bindView(final Activity activity,final int res) {
        return LazyKt.lazy(activity.findViewById(res));
    }
}

File: kotlinlib/src/main/java/com/example/lib/TestAct.kt
package com.example.lib

import android.app.Activity
import android.view.View
import android.widget.Button

class TestAct : Activity() {
    private val mRootView: Button by bindView(R.layout.activity_main)

    // private Lazy<View> bindView(final Activity activity, @IdRes final int res) {
    //        return LazyKt.lazy(() -> activity.findViewById(res));
    //    }
    private fun <T : View> Activity.bindView(res: Int): Lazy<T> {
        return lazy { findViewById(res) }
    }
}